spec/latest/json-ld/index.html

```diff
@@ -3790,6 +3741,9 @@
       <code>@nest</code> mapping. When expanding, the values of a property
       expanding to <code>@nest</code> are treated as if they were contained
       within the enclosing <a>node object</a> directly.</li>
+    <li>The JSON syntax has been abstracted into an <a>internal representation</a>
+      to allow for other seriazation formats that are functionally equivalent
```

Comment:
typo: "seriazation" => "serialization"

Could maybe just say "serializations" instead of "serialization formats" to save a word :)